spec/requests/memberships_controller_spec.rb

```diff
@@ -285,4 +285,25 @@
       end
     end
   end
+
+  describe '#withdraw' do
+    include_context 'current_user is a member'
+    context 'when membership has DEFAULT post' do
+      let(:membership) { create(:membership, :newbie) }
+      it 'withdraws the membership' do
+        post "/groups/#{membership.group.id}/memberships/#{membership.id}/withdraw", xhr: true
+
+        expect(response).to redirect_to group_path(group)
+      end
+    end
+
+    context 'when membership has no DEFAULT post' do
+      let(:membership) { create(:membership) }
+      it 'it returns forbidden' do
```

Comment:
I sense redundancy here

```suggestion
      it 'returns forbidden' do
```